gulpfile.js

```diff
@@ -354,10 +354,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.8'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.9.1'))
+    .pipe(replace('[ios.current-app-version]', '2.10.1'))
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.9.0'))
+    .pipe(replace('[android.current-app-version]', '2.10.0'))
```

Comment:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.10.1'))
```

See https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.10.1-RC1